src/index.ts

```diff
@@ -33,7 +33,18 @@ const isDevelopmentVersion = (version: string) => {
 	return /(\.|\+|-)dev/.test(version);
 };
 
-export const compare = memoize((versionA: string, versionB: string) => {
+export const compare = memoize((versionA: string | null, versionB: string | null) => {
+	if (versionA === null && versionB === null) {
+		return 0;
+	}
+	if (versionA === null && versionB !== null) {
+		return -1;
+	}
+	if (versionA !== null && versionB === null) {
+		return 1;
+	}
+	versionA = <string>versionA;
+	versionB = <string>versionB;
```

Comment:
If you've got strict nulls on, you should be able to avoid this cast. TypeScript will remove types from union types like this if it knows you've checked for them and already returned above.